packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -151,22 +155,59 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
         }
     }
 
-    private setTruncationState() {
-        if (!this.props.detectTruncation || this.props.showPopover !== TruncatedPopoverMode.WHEN_TRUNCATED) {
+    private setTruncationState(prevProps?: ITruncatedFormatProps) {
+        const {
+            detectTruncation,
+            parentCellHeight,
+            parentCellWidth,
+            showPopover: popoverMode,
+        } = this.props;
+
+        // this is where we get our savings from the injected parent-size props.
+        // if they haven't changed between updates, then no need to remeasure!
+        if (!detectTruncation
```

Comment:
This doesn't quite work -- the parent heights and widths may not have changed, but the _content_ could have. You still need to remeasure if anything changed. We can't really get savings this way.